server/src/main/kotlin/net/horizonsend/ion/server/command/starship/BlueprintCommand.kt

```diff
@@ -63,9 +63,9 @@ object BlueprintCommand : net.horizonsend.ion.server.command.SLCommand() {
 		failIf(name.length !in 2..50) {
 			"Name length is ${name.length}, must be between 2 and 50"
 		}
-		failIf(!name.replace('-', ' ').replace('_', ' ').isAlphanumeric()) {
-			"Name must only contain letters, numbers, and - or _"
-		}
+		failIf(name.replace('-', ' ').replace('_',' ').replace(" ","").isAlphanumeric()){
```

Comment:
why replace `'_'` with `' '` and then immediately replace `' '` with `''`?

Reply:
cause you can't have an empty character literal, it throws an error